code/extension/ShareThisSTE.php

```diff
@@ -91,50 +124,67 @@ public function ShowShareIcons()
         }
     }
 
-    public function ShareIcons()
+    /**
+     * Get the sharing icons
+     */
+    public function getShareIcons()
     {
         $bookmarks = $this->makeBookmarks('IncludeThisIcon');
         return $this->makeShareIcons($bookmarks);
     }
 
+    /**
+     * Grabbing front end dependencies for the expanded sharing list with some extra
+     * functionality
+     */
     public function ShareAllExpandedList()
     {
-        Requirements::javascript(FRAMEWORK_DIR . '/thirdparty/jquery/jquery.js');
+        Requirements::javascript(SS_SHARETHIS_DIR . '/node_modules/jquery/dist/jquery.min.js');
```

Comment:
I don't see this folder unless I do a yarn install inside this module folder. It probably makes sense to instead pull in jquery from silverstripe/admin/thirdparty/jquery which should be exposed in the public resources folder.